cpp/src/aztec/honk/sumcheck/polynomials/multivariates.hpp

```diff
@@ -104,9 +104,14 @@ template <class FF_, size_t num_polys, size_t num_vars> class Multivariates {
 
     std::array<FF, num_polys> batch_evaluate(std::array<FF, num_vars> input)
     {
-        // TODO(Cody): IOU implementation.
+        // TODO(Cody): these just get extracted from the folded multivariates
+        // For now, at least initialize properly.
         static_cast<void>(input);
-        return { { 1 } };
+        std::array<FF, num_polys> result;
+        for (auto& elt : result) {
+            elt = 1;
```

Comment:
`elt`?

Reply:
"element". But you're anti-abbreviation, I can change.

Reply:
hah I genuinely couldn't decipher that one

Reply:
This is a clear sign you were in applied math.